build/AzurePipelinesTemplates/WindowsAppSDK-BuildBinaries-Steps.yml

```diff
@@ -0,0 +1,123 @@
+parameters:
+- name: "SignOutput"
+  type: boolean
+  default: False
+
+steps:
+- task: NuGetToolInstaller@1
+
+- task: NuGetAuthenticate@1
+  inputs:
+    nuGetServiceConnections: 'TelemetryInternal'
+
+- task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+  displayName: 'NuGet restore of packages'
+  inputs:
+    command: 'custom'
+    arguments: 'restore dev\WindowsAppRuntime_Insights\packages.config -ConfigFile NuGet.config -PackagesDirectory dev\WindowsAppRuntime_Insights\packages'
+
+- template: WindowsAppSDK-SetupBuildEnvironment-Steps.yml
+
+- task: NuGetAuthenticate@1
+  inputs:
+    nuGetServiceConnections: 'ProjectReunionInternal'
+
+  # Copy MSIX license installation header into the correct source location.
+  # Restore transport package dependencies. This is only enbaled in release-signed builds.
+- task: PowerShell@2
+  name: ConvertVersionDetailsToPackageConfig
+  displayName: "Convert VersionDetails To PackageConfig"
+  inputs:
+    filePath: '$(REPOROOT)\build\Scripts\ConvertVersionDetailsToPackageConfig.ps1'
+    arguments: -versionDetailsPath '$(REPOROOT)\eng\Version.Details.xml' -packageConfigPath '$(REPOROOT)\build\packages.config'
+
+- task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+  displayName: RestoreNuGetPackages
+  inputs:
+    restoreSolution: build/packages.config
+    feedsToUse: config
+    nugetConfigPath: NuGet.config
+    restoreDirectory: packages
+
+- task: CopyFiles@2
+  displayName: 'Extract license header to source location'
+  inputs:
+    SourceFolder: '$(REPOROOT)\build\packages\$(AppLicensingInternalPackageName).$(AppLicensingInternalPackageVersion)\src'
+    Contents: |
+      *.h
+    TargetFolder: '$(REPOROOT)\dev\Licensing'
+    flattenFolders: false
+    overWrite: true
+
+- task: PowerShell@2
+  name: BuildBinaries
+  inputs:
+    filePath: 'BuildAll.ps1'
```

Comment:
My understanding is that this change will prevent us from running PREFast for MRT. Is this intended? Or OneBranch has "magic" to make it work?

Reply:
Yes all the sdl analysis should be injected for us.

Reply:
My understanding is that PREFast is not injected for us automatically.
[SDL (Static Analysis) for Containerized Workflows](/OneBranch/Security,-Compliance,-Governance-and-Policies/SDL-\(Static-Analysis\)-for-Containerized-Workflows)

Reply:
Synced Offline with Alex. We will bring back 'Guardian: PreFast@3' to the MRT Stage